### PR TITLE
Add Skill section

### LINKED ISSUE #4: Add skills
Would be great to highlight what programming languages people are comfortable to write. Maybe include an image too like the experience option.

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
Fixes #4
## Changes
A skill section is added based on catgorization of the skill itself, like Programming skills, framework knowledge and Interpersonal skills in the form of cards.

## How Has This Been Tested?
- This has been tested locally in the systems and in the docker environment with no errors, warnings etc on the console. 
- This has been tested for responsiveness for multiple screens.

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] New feature and improvements (non-breaking change which adds/improves functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code Refactoring (internal change which is not user facing)
- [ ] Documentation
- [ ] Test, CI, or build

## Checklist

- [x] Coding style of this project is followed
- [ ] Change in documentation is required
- [ ] Documentation has been updated accordingly 
- [ ] Unit tests have been added to cover code change

### CODE REVIEW

**Review by Korusuke — CHANGES_REQUESTED** [review]
Hey @Abhishek-kumar09, thanks for submitting the PR!

The code itself looks solid but I am a bit concerned about asset images, is there way to get these from some third party service and not have them in the template?

cause if someone wants to add their own skills they will have to go looking for images and resize that and everything.

File: Gemfile.lock [hunk 1]
```diff
@@ -69,4 +70,4 @@ DEPENDENCIES
   webrick (~> 1.7)
 
 BUNDLED WITH
-   2.1.4
+   2.2.2
```
[the remark this answers is not in the document]
Comment (Abhishek-kumar09) [reply Gemfile.lock]: @Korusuke, This changes are generated automatically when ran on docker, left here by mistake.
Reverted. Done.

**Comment by Abhishek-kumar09** [comment]
>  am a bit concerned about asset images, is there way to get these from some third party service and not have them in the template?

yes, I have just added the fontawesome and devIcons to fetch regular and development related icons respectively.

File: _data/skills.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      
+    - skill: C++
+      profeciency: Beginner
+      icon: devicon-c-plain
+
+    - skill: Java
+      profeciency: Intermediate
+      icon: devicon-java-plain
+
+#frameworks section
```
Comment (Korusuke) [inline _data/skills.yml:19]: nitpick: add a space after `#`

File: _data/skills.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    - skill: Django
+      profeciency: Advanced
+      icon: devicon-django-plain
+      
+    - skill: Express.js
+      profeciency: Beginner
+      icon: devicon-express-original
+
+# #interpersonal section
```
Comment (Korusuke) [inline _data/skills.yml:34]: nitpick: only 1 `# ` should be there for comments

**Review by Korusuke — CHANGES_REQUESTED** [review]
Woahh!! The skills section looks amazing!

Just had few nitpicks and then this PR is ready to be merged.

I also had one last doubt, currently there 3 in each category but what if I have more how would it look? can u post a screenshot with 6 3 2.

**Comment by Abhishek-kumar09** [comment]
Wow, it broke:
image

Will fix it.

File: _data/skills.yml (hunk 2 above)
Comment (Abhishek-kumar09) [reply _data/skills.yml]: Done.

File: _data/skills.yml (hunk 3 above)
Comment (Abhishek-kumar09) [reply _data/skills.yml]: Done, these are blunders to be honest.

**Comment by Abhishek-kumar09** [comment]
> I also had one last doubt, currently there 3 in each category but what if I have more how would it look? can u post a screenshot with 6 3 2.

This is how it will look.
Screenshot_2021-02-04 Personal Portfolio

**Comment by Abhishek-kumar09** [comment]
@Korusuke Please let me know if you have some other suggestions.

**Comment by Korusuke** [comment]
it looks fine to me, sorry I forgot to merge....merging now!

**Pull request merged**